src/tools/mesh_tester.cpp

```diff
@@ -40,20 +40,20 @@
 // RAJA policies
 #include "axom/mint/execution/internal/structured_exec.hpp"
 
-#if defined (AXOM_USE_RAJA)
-  using seq_exec = axom::SEQ_EXEC;
+#if defined(AXOM_USE_RAJA)
+using seq_exec = axom::SEQ_EXEC;
 
   #if defined(AXOM_USE_OPENMP)
-    using omp_exec = axom::OMP_EXEC;
+using omp_exec = axom::OMP_EXEC;
```

Comment:
This was one of the only cases I found where clang-format did something that I don't like and I couldn't find a workaround.

Specifically, the nested compiler defines are indented, but the code lines are not.

Overall, I think we should nest our compiler defines/pragmas since it dramatically improves code readability.

I didn't find other examples and I think we can code around this (or disable formatting) if it bothers us.